### DISCUSSION TITLE
How to change <pre> tag styles

### DISCUSSION THREAD

**Discussion by nimatrazmjo** [remarkjs/remark#1004] [General] [answered]
Hello guys,
I have used `react-markdown` package in my project, and I am wondering is there any way to change the default `<pre>` tag color

# from:

<img width="668" alt="Screen Shot 2022-06-09 at 7 05 56 AM" src="https://user-images.githubusercontent.com/3712526/172752348-298d71bb-a809-444d-9f24-912e005bf0b8.png">

# to:

<img width="693" alt="Screen Shot 2022-06-09 at 7 09 18 AM" src="https://user-images.githubusercontent.com/3712526/172752511-81116ac9-e570-4e3f-8101-fc41eebd6958.png">

Thanks in advance

**Comment by ChristianMurphy** [comment, marked as answer]
Using just `react-markdown` CSS can be used to change colors.

If you additionally want syntax highlighting with specific themes, see https://github.com/remarkjs/react-markdown#use-custom-components-syntax-highlight
